examples/feature_engineering.py

```diff
@@ -0,0 +1,333 @@
+"""
+This example demonstrates the use of the Prefect platform for machine learning.
+Data is downloaded from a URL, extracted, and loaded into Numpy/Pandas.
+Then a series of mathematical operations are performed on the columns, to create
+a feature set from the raw data. Finally, the features are stitched together and
+passed into a Principal Components Analysis for dimensionality reduction.
+
+The Flow makes use of a number of Prefect features, including Parameters,
+the set_dependencies method for Tasks that don't pass data, task mapping,
+and custom Task classes.
+"""
+
+import os
+import pathlib
+import zipfile
+import pandas as pd
+import numpy as np
+from urllib.request import urlretrieve
+
+from prefect import Flow, Task, Parameter, task
+from prefect import unmapped
+from prefect.tasks.core.constants import Constant
```

Comment:
Now that #896 is merged, if you update with the `master` branch you should be able to remove this use of `Constant` now